geometry_msgs/Pose2D pose
int8 dim
int8 color